### DISCUSSION TITLE
Field - doesn't respect the "required" attr

### DISCUSSION THREAD

**Discussion by xeinebiu** [chakra-ui/ark#2986] [General]
### Description

![image](https://github.com/user-attachments/assets/10f9d1bd-1614-48fd-9005-4bd8c4f4fb53)

As seen on the image above, the required attr has no effect to display the asterisk

### Link to Reproduction (or Detailed Explanation)

None

### Steps to Reproduce

````tsx
const Main: FC = () => {
    return (
        <StrictMode>
            <NiceModal.Provider>
                <Content/>
            </NiceModal.Provider>
            <GlobalToastProvider/>
        </StrictMode>
    );
};

const Content: FC = () => {
    return (
      <Field.Root required>
        <Field.Label>Label</Field.Label>
        <Field.Input  required />
        <Field.HelperText>Some additional Info</Field.HelperText>
        <Field.ErrorText>Error Info</Field.ErrorText>
      </Field.Root>
    )
};

const root = ReactDOM.createRoot(
    document.getElementById('root') as HTMLElement,
);
root.render(<Main/>);
````

### Ark UI Version

4.1.0

### Framework

- [X] React
- [ ] Solid
- [ ] Vue

**Comment by segunadebayo** [comment]
Rendering an asterisk is out of scope of Ark UI, although we’re considering adding a dedicated component for this.

In the meantime, please use the useFieldContext to read the required and render an asterisk in your project.

**Reply by segunadebayo** [reply]
Added support for this in Ark. Should be available in the next release

**Comment by segunadebayo** [comment]
Closing — Field now properly passes `required` through to all control elements. This has been fixed.
